### ISSUE TITLE
Quest [Mending Fences] 12915 Buggy

### ISSUE THREAD

**Issue by Pestilenz** [WoWManiaUK/Blackwing-Lair#4293]
At Quest Mending Fences Thorim send you to eastern Dun Niffelem to slay 5 Stormforged Iron Giants and Fjorn.
Maybe it is a Phasing problem, but the adds aren´t there.
No need of use of the item you got, that supports you only.

https://www.youtube.com/watch?v=Yvp-imxsS7c

**Comment by Echinaceax** [comment]
No issue found @Pestilenz 
What race is your char?

![no issue](https://user-images.githubusercontent.com/39439201/77233466-df66fb00-6ba7-11ea-8aaa-5eb0308f594b.jpg)

Make sure your up to date with your launcher (run launcher as admin)

**Comment by Echinaceax** [comment]
@Rushor Quest works and completed.
No quest bug - Possible char phasing issue only

**Comment by Pestilenz** [comment]
neither it worked with my undead dk, nor with my blood elf mage.
And is did the whole q chain till that point

Am Sa., 21. März 2020 um 19:14 Uhr schrieb Echinaceax <
notifications@github.com>:

**Comment by Echinaceax** [comment]
Yes thats why i think its a char bug,  if you have it on your Belf too then maybe its a patching or launcher issue because i didnt have any issue with my test Bloodelf priest

**Comment by Pestilenz** [comment]
I have the original update reign launcher, use it, and on both chars just
then got phased when zinius ported me to himself

Am Mi., 25. März 2020 um 19:23 Uhr schrieb Echinaceax <
notifications@github.com>:

**Comment by Echinaceax** [comment]
Ok @Pestilenz then im sure @Rushor will find out what the issue is, As a tester i just know I had no issues, ive wondering if any other player has has the same issue as you, if they had it would have been reported earlier

**Comment by Pestilenz** [comment]
maybe when i start q chain i get phased, but not phased back? so "normal"
is last phase, and beginning the q chain phases you in 1 of the other
phases?
you have completed the K3-Brunhildar-Altar of Storms (when i remember
right, left Side Storm Peaks Tower) q chain?

Am Mi., 25. März 2020 um 20:50 Uhr schrieb Echinaceax <
notifications@github.com>:

**Comment by Echinaceax** [comment]
Exactly what i think has happened, ive asked players in discord if they have had the same issue as you
